internal/common/common.go

```diff
@@ -109,3 +110,15 @@ func HostArch() string {
 		return runtime.GOARCH
 	}
 }
+
+// SortedMapKeys returns a sorted list of the map keys
+// Only works on maps with string as the key
+func SortedMapKeys(m map[string]any) []string {
+	keys := []string{}
```

Comment:
```suggestion
	keys := maps.Keys(m)
```
…and removing the next 3 lines, should be working as well (since go 1.21) right?

Reply:
Some of this code is starting to age... :)

Reply:
Oh, right. maps was added in 1.21 but Keys is only in 1.23